app/routes/ags_export_by_polygon.py

```diff
@@ -0,0 +1,106 @@
+import requests
+
+from fastapi import APIRouter, Request
+from fastapi.exceptions import HTTPException
+
+import shapely
+
+from requests.exceptions import Timeout, ConnectionError, HTTPError
+
+from app.model.schema import BoreholeCountResponse
+from app.model.queries import polygon_query, count_only_query
+from . ags_export import ags_export
+from . utils import (get_request_url, ags_export_responses, BOREHOLE_INDEX_URL,
+                     BOREHOLE_EXPORT_LIMIT, BOREHOLE_EXPORT_URL)
+
+router = APIRouter()
+
+
+@router.get("/ags_export_by_polygon/",
+            tags=["ags_export_by_polygon"],
+            summary="Export a number of boreholes in .ags format in a polygon",
+            description=("Export a number of boreholes in .ags format from AGS data "
+                         "held by the National Geoscience Data Centre, using a"
+                         " polygon using Well-Known-Text."),
+            response_model=BoreholeCountResponse,
+            responses=ags_export_responses)
+def ags_export_by_polygon(polygon: str = polygon_query,
+                          count_only: bool = count_only_query,
+                          request: Request = None):
+    """
+    Export the boreholes in .ags format from AGS data held by the National Geoscience Data Centre,
+    that are bounded by the polygon. If there are more than 50 boreholes return an error
+    :param polygon: A polygon in Well Known Text.
+    :type polygon: str
+    :param count_only: The format to return the validation results in. Options are "text" or "json".
```

Comment:
The docstring for ags_export_by_polygon seems to have a copy-paste error in the description of the count_only parameter.